prototype/consensus/debug_handlers.go

```diff
@@ -0,0 +1,111 @@
+package consensus
+
+import (
+	"fmt"
+	"google.golang.org/protobuf/types/known/anypb"
+	"log"
+	"pocket/consensus/dkg"
+	"pocket/consensus/types"
+	shared_types "pocket/shared/types"
+)
+
+func (m *ConsensusModule) handleDebugMessage(message *DebugMessage) {
+	switch message.Action {
+	case TriggerNextView:
+		m.handleTriggerNextView(message)
+	case SendTx:
+		m.handleSendTx(message)
+	case TriggerDKG:
+		m.handleTriggerDKG(message)
+	case TogglePaceMakerManualMode:
+		m.handleTogglePaceMakerManualMode(message)
+	case ResetToGenesis:
+		m.resetToGenesis(message)
+	case PrintNodeState:
+		types.GetTestState().PrintGlobalState()
+		m.printNodeState(message)
+	default:
+		log.Fatalf("Unsupported debug message: %s \n", StepToString[Step(message.Action)])
+	}
+}
+
+func (m *ConsensusModule) handleSendTx(debugMessage *DebugMessage) {
+	// convert to consensus message
+	//txMessage := &TxWrapperMessage{
+	//	Data: debugMessage.Payload,
+	//}
+	// create the event
+	event := shared_types.Event{
+		SourceModule: shared_types.CONSENSUS_MODULE,
+		PocketTopic:  string(shared_types.UTILITY_TX_MESSAGE),
+	}
+	// convert to network message
+	consensusProtoMsg := &types.Message{
+		Data: debugMessage.Payload,
+	}
+
+	anyProto, err := anypb.New(consensusProtoMsg)
+	if err != nil {
+		m.nodeLogError("Error encoding any proto: " + err.Error())
+		panic("^")
```

Comment:
While I was looking at this, it came to me that I also was thinking about what to do on panic behaviors, what is our logic for recovery and all of that... an epic on its own

Reply:
ACK. We'll design something like this correctly in the future.